analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -66,7 +66,17 @@ import okhttp3.Request
 const val PIP_VERSION = "18.0"
 
 const val PIPDEPTREE_VERSION = "0.13.0"
-val PIPDEPTREE_DEPENDENCIES = arrayOf("pipdeptree", "setuptools", "wheel")
+private val IGNORED_DEPENDENCIES = mapOf(
+    "pipdeptree" to "", // A dependency of pipdeptree itself.
+    "pkg-resources" to "0.0.0", // Added by a bug with some Ubuntu distributions.
+    "setuptools" to "", // A dependency of pipdeptree itself.
+    "wheel" to "" // A dependency of pipdeptree itself.
+)
+
+private fun isDependencyIgnored(name: String, version: String): Boolean =
```

Comment:
Hmm, I'm probably again becoming nitty, but this function does not tell you whether a dependency *is* ignored (e.g. altready by `pip`), but whether it *should* be ignored (in our post-processing). Maybe rename bot the map and function to something involving "inherent", "bogus" or "phony"? Like `PHONY_DEPENDENCIES` and `isPhonyDepedency`?

Reply:
> but whether it _should_ be ignored

I also didn't like the name 100% but I had no idea I really liked so I left it as is because I thought it is good enough. 

...I also thought about 'shouldIgnoreDependency' which I didn't like.